example/ios/Runner/DeepLinkPlugin.swift

```diff
@@ -118,34 +120,34 @@ private extension DeepLinkPlugin {
             return false
         }
         
-        // Check required parameters
-        let requiredParams = ["sessionId", "nonce", "timestamp", "signature"]
+        // Build param dictionary
         let paramDict = Dictionary(uniqueKeysWithValues: queryItems.map { ($0.name, $0.value) })
-        
+
+        // For new start-app flow: only require sessionId and nonce for now
+        if url.host == "passport-issuer.staging.yivi.app" && url.path.hasPrefix("/start-app") {
+            guard let sessionId = paramDict["sessionId"], !sessionId.isEmpty,
+                  let nonce = paramDict["nonce"], !nonce.isEmpty else { return false }
+            // Light format checks
+            guard sessionId.range(of: "[0-9A-Fa-f-]{10,}", options: .regularExpression) != nil else { return false }
+            guard nonce.range(of: "[A-Za-z0-9-_+/]+=*", options: .regularExpression) != nil else { return false }
+            return true
+        }
+
+        // Legacy validate flow: require all parameters
+        let requiredParams = ["sessionId", "nonce", "timestamp", "signature"]
         for param in requiredParams {
-            guard let value = paramDict[param], !value.isEmpty else {
-                return false
-            }
+            guard let value = paramDict[param], !value.isEmpty else { return false }
         }
-        
+
         // Validate sessionId format (UUID)
-        guard let sessionId = paramDict["sessionId"],
-              isValidUUID(sessionId) else {
-            return false
-        }
-        
+        guard let sessionId = paramDict["sessionId"], isValidUUID(sessionId) else { return false }
+
         // Validate timestamp format
-        guard let timestampString = paramDict["timestamp"],
-              let _ = Int64(timestampString) else {
-            return false
-        }
-        
+        guard let timestampString = paramDict["timestamp"], Int64(timestampString) != nil else { return false }
+
         // Basic nonce validation (should be Base64)
-        guard let nonce = paramDict["nonce"],
-              isValidBase64(nonce) else {
-            return false
-        }
-        
+        guard let nonce = paramDict["nonce"], isValidBase64(nonce) else { return false }
```

Comment:
In the PR comment you mention: `the nonce should be a hexadecimal representation of a byte[8]` but there is no check on the nonce size, is that correct?

Reply:
Fixed that